src/Bounds.cpp

```diff
@@ -1468,6 +1480,7 @@ class Bounds : public IRVisitor {
             }
         } else if (op->args.size() == 1 &&
                    (op->is_intrinsic(Call::round) ||
+                    op->is_intrinsic(Call::strict_float) ||
```

Comment:
There's going to be a merge conflict here because Call::saturating_cast is in the same category. Probably should add it in this PR in case the other one doesn't go in and we revert the u32 -> i32 cast change.

Reply:
I take it back! saturating_cast doesn't belong here.